pretixprint/app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="eu.pretix.pretixprint">
```

Comment:
Is that removal on purpose?

Reply:
yep, Android Studio removed that automatically, it is now replaced by the `applicationId` in build.gradle